plugins/stk/mallets/mallets.cpp

```diff
@@ -335,6 +325,18 @@ malletsInstrumentView::malletsInstrumentView( malletsInstrument * _instrument,
 	m_spreadKnob->setLabel( tr( "Spread" ) );
 	m_spreadKnob->move( 190, 140 );
 	m_spreadKnob->setHintText( tr( "Spread:" ), "" );
+
+	// try to inform user about missing Stk-installation
+	if( ( !QDir( ConfigManager::inst()->stkDir() ).exists() ||
+		  !QFileInfo( ConfigManager::inst()->stkDir() + QDir::separator()
+						  + "sinewave.raw" ).exists() ) && Engine::hasGUI() )
+	{
```

Comment:
Is there a way to use `m_filesMissing` from the instrument instead of replicating this logic?

-Tres